### ISSUE TITLE
gh-md-toc broken?

### ISSUE THREAD

**Issue by icct-galvarez** [theicct/roadmap-doc#3]
Realized today that the v2.4 documentation (https://github.com/theicct/roadmap-doc/commit/4a7bad02ec90fb85b69587cf7269d85c439d51f8) doesn't have a table of contents . I re-ran `bash copy_doc.sh` and it runs fine, but just doesn't add a ToC as expected. I thought that maybe it was because I changed the image on the title page (finally removing the grey background), but I was able to reproduce the issue with an old version of the word doc (at v2.3) which had previously worked. 

So it appears that something has broken with the ToC generator. And looking at their GH repo, it looks like we are not the only one having this problem: https://github.com/ekalinin/github-markdown-toc/issues/146. 

Need to investigate further and see if there is a change I can make locally, or if the package is currently unusuable. For now, I manually added the ToC to v2.4 (476367d).

**Comment by icct-galvarez** [comment]
Discussed with Arijit and he agreed it is worth spending a few hours to dig around and see if we can get this working, since manually keeping the ToC up to date across all the different documentation pages is not ideal.

**Comment by icct-galvarez** [comment]
Still broken when I published the v2.5 docs today, but don't have time to investigate right now.

**Comment by icct-galvarez** [comment]
Resolved in v0.10.0. Just [re-install](https://github.com/ekalinin/github-markdown-toc?tab=readme-ov-file#installation) the latest version:

```
curl https://raw.githubusercontent.com/ekalinin/github-markdown-toc/master/gh-md-toc -o gh-md-toc
chmod a+x gh-md-toc
```

**Comment by icct-galvarez** [comment]
Noticed that gh-md-toc was being manually tracked in the repo (added in 47e7c7e). Not good practice to track externally maintained dependencies in our repo, so removed it in 1b71ed8.
